### ISSUE TITLE
star component nvda issue

### ISSUE THREAD

**Issue by scottaohara** [scottaohara/a11y_styled_form_controls#30]
the star radios do not announce their accessible name when using nvda virtual cursor outside of forms mode.

fix thought to be removing the aria-label from the radio buttons, and removing the visibility: hidden from the hidden labels text that pops into view.  removing these had no positive effects for nvda+ff, and had no detrimental effects for other screen readers, so they can be left off.

this may be a firefox only issue as it announces correctly with chrome and IE11.  tested with both esr and nightly v62
